x/docs/internal/keeper/querier_test.go

```diff
@@ -4,20 +4,20 @@ import (
 	"testing"
 
 	"github.com/commercionetwork/commercionetwork/types"
+	keys "github.com/commercionetwork/commercionetwork/x/docs/internal/types"
```

Comment:
I think we can avoid naming this import `keys` and just leave the blank `types`, or are there any problem?

Reply:
There are conflict with the import above that, in fact they have the same name.
So i use the keywork keys also to rember me in which files prefixes is.

Reply:
I've moved both `Document` and `DocumentReceipt` inside the `x/docs/internal/types` folder, and I've created their aliases inside the `alias.go` file so now this has been fixed.